eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -318,7 +327,7 @@ function Remove-EmptySections {
     $parsedSections = $ChangeLogEntry.Sections
     $sanitizedReleaseContent = New-Object System.Collections.ArrayList(,$releaseContent)
   
-    foreach ($key in @($parsedSections.Key)) 
+    foreach ($key in @($parsedSections.Keys)) 
     {
       if ([System.String]::IsNullOrWhiteSpace($parsedSections[$key]))
```

Comment:
Do we need to trim lines here? Does whitespace include newlines?

Reply:
Yes that handles empty newlines.